librz/bin/bin.c

```diff
@@ -673,6 +675,33 @@ RZ_API RZ_BORROW RzBinMap *rz_bin_object_get_map_at(RZ_NONNULL RzBinObject *o, u
 	return NULL;
 }
 
+/**
+ * \brief Find the binary symbol at offset \p off.
+ *
+ * This function returns the binary symbol that contains offset \p off.
+ * It iterates over the list of symbols and directly compares the offset with the virtual or physical address.
```

Comment:
I recommend removing this line, especially since soon symbols will be converted into RzPVector.

Reply:
Sure, can you check my most recent commit I haven't removed this line as I saw the comment after i pushed it.